src/main/java/com/iota/iri/storage/LocalSnapshotsPersistenceProvider.java

```diff
@@ -0,0 +1,68 @@
+package com.iota.iri.storage;
+
+import com.iota.iri.utils.Pair;
+
+import java.util.List;
+
+/**
+ * Abstraction for localsnapshots-db persistence provider.
+ */
+public class LocalSnapshotsPersistenceProvider {
+    private PersistenceProvider provider;
+
+    /**
+     * Initialize the provider
+     * @throws Exception
+     */
+    public void init() throws Exception{
+        provider.init();
+    }
+
+
+    public void injectProvider(PersistenceProvider persistenceProvider){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4082603)